blockly-rtc/webpack.config.js

```diff
@@ -18,7 +18,7 @@
 
 /**
  * @fileoverview Webpack configuration file.
- * @author samelh@google.com (Sam El-Husseini)
+ * @author navil@google.com (Sam El-Husseini)
```

Comment:
Realizing I did not change the author name. I will upload the fix in a following commit along with any other changes requested.